examples/flows/chat/chat-basic-code-first/flow.py

```diff
@@ -25,7 +25,16 @@ class Result:
 @trace
 def flow_entry(question: str='What is ChatGPT?', chat_history: list = []) -> Result:
     """Flow entry function."""
+    from promptflow._sdk._configuration import Configuration
+
     prompt = load_prompt("chat.jinja2", question, chat_history)
+    config = Configuration.get_instance()
+    # TODO: create your own config.json
+    workspace_config = config._get_workspace_from_config(path="./config.json")
```

Comment:
we need to ensure the sample works for local & cloud

Reply:
Offline synced, since this sample is part of prompt flow managed connection experience. We'll need to finalize the design

Reply:
If we allow user to get connections on his own, there will be a ask for runtime to set connection provide to current worksapce.